libs/log/tm_logger.go

```diff
@@ -59,6 +59,15 @@ func (l *tmLogger) Info(msg string, keyvals ...interface{}) {
 	}
 }
 
+func (l *tmLogger) IsDebugging() bool {
+	lWithLevel := kitlevel.Debug(l.srcLogger)
+
+	if err := kitlog.With(lWithLevel, msgKey, "").Log(""); err != nil {
+		return true
+	}
+	return false
```

Comment:
Is this not something that can be memoized?

Reply:
technically you _can change_ log level at runtime... I don't know if _anyone has ever tried_

Reply:
AFAIK we don't actually offer a way to do that. The logger is configured via the command line or config file, both of which require a process restart to read, so memoization should be a-ok.

Reply:
Cool.. I'll consider memoization for the tm_logger

Reply:
I'll defer tm_logger memoization for a future effort :)